code/modules/jobs/access.dm

```diff
@@ -31,6 +33,13 @@
 			return TRUE
 	return FALSE
 
+/obj/proc/handle_buckled_access(mob/accessor)
+	. = FALSE
+	// check if someone riding on / buckled to them has access
+	for(var/mob/living/buckled in accessor.buckled_mobs)
```

Comment:
although it won't happen, let's put a infinite loop failsafe here: `if(src == buckled) continue`